apps/tlon-web/src/state/chat/chat.ts

```diff
@@ -555,7 +556,7 @@ function infiniteDMsUpdater(queryKey: QueryKey, data: WritDiff | WritResponse) {
 
             const newReplies = {
               ...prevReplies,
-              [formatUd(bigInt(reply.delta.add.time!))]: newReply,
```

Comment:
we need to use this in cases where we're hearing replies, otherwise if optimistic use memo.sent